web-components/src/components/usa-link/link.stories.js

```diff
@@ -0,0 +1,21 @@
+import "./index";
+
+import { html } from 'lit';
```

Comment:
Use double quotes for consistency
```suggestion
import { html } from "lit";
```

Reply:
Good point. Fixed in 41ec42b.